### ISSUE TITLE
Ingame ToolTip is late for a while

### ISSUE THREAD

**Issue by White-Raven** [RaiderIO/raiderio-addon#171]
Tried uninstalling the addon and pulling it freshed, cleared cache, and the ingame tool tip caps at 5 kills of each boss since I reached 5 kill of each at different weeks, doesn't seem to want to go higher.
https://imgur.com/MgubQFd

And as you guessed it been for a while
https://imgur.com/ANO4UtA

And it's quite annoying knowing I use the desktop client, have linked my bnet account, and that raid.io is ignored in curse's client.
Long story short, the setup is just fine.
